src/main/kotlin/org/malachite/estella/task/api/TaskController.kt

```diff
@@ -3,67 +3,85 @@ package org.malachite.estella.task.api
 import org.malachite.estella.commons.EStellaHeaders
 import org.malachite.estella.commons.Message
 import org.malachite.estella.commons.OwnResponses
+import org.malachite.estella.commons.UnauthenticatedException
 import org.malachite.estella.process.domain.TaskDto
 import org.malachite.estella.process.domain.TaskTestCaseDto
 import org.malachite.estella.process.domain.toTask
+import org.malachite.estella.services.OrganizationService
 import org.malachite.estella.services.RecruitmentProcessService
 import org.malachite.estella.services.TaskService
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 import org.springframework.transaction.annotation.Transactional
 import org.springframework.web.bind.annotation.*
+import javax.websocket.server.PathParam
+
 
 @RestController
 @RequestMapping("/api/tasks")
 class TaskController(
     @Autowired private val taskService: TaskService,
-    @Autowired private val processService: RecruitmentProcessService
+    @Autowired private val organizationService: OrganizationService,
 ) {
 
     @CrossOrigin
     @Transactional
     @GetMapping
-    fun getTaskByRecruitmentProcess(@RequestParam(name = "process") processId: Int): ResponseEntity<List<TaskDto>> =
-        taskService
-            .getTasksByRecruitmentProcess(processId)
+    fun getTaskByRecruitmentProcess(
+        @RequestParam("owner") organizationUuid: String,
+        @RequestHeader(EStellaHeaders.passwordHeader) password: String
+    ): ResponseEntity<List<TaskDto>> =
+        taskService.checkDevPassword(organizationUuid, password)
+            .also { println("Tutaj jebło") }
+            .getTasksByOrganizationUuid(organizationUuid)
+            .also { println("Tutaj jebło x2") }
             .let { ResponseEntity.ok(it) }
 
     @Deprecated(message = "Wasn't tested yet - unnecessary now - to be implemented and tested in ES-17 epic")
     @CrossOrigin
     @Transactional
     @GetMapping("/{taskId}")
-    fun getTaskById(@PathVariable taskId: Int) = ResponseEntity(Message("Not Implemented"), HttpStatus.NOT_IMPLEMENTED) /*taskService.getTaskById(taskId)
+    fun getTaskById(
```

Comment:
jeśli są do tego już testy i jest to ogarnięte, to możesz wywalić @deprecated i zakomentowany kod